tello_driver/CMakeLists.txt

```diff
@@ -29,44 +29,46 @@ set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -pthread")
 # Find packages
 find_package(ament_cmake REQUIRED)
 find_package(camera_calibration_parsers REQUIRED)
+find_package(class_loader REQUIRED)
 find_package(cv_bridge REQUIRED)
 find_package(OpenCV REQUIRED)
 find_package(rclcpp REQUIRED)
+find_package(rclcpp_components REQUIRED)
 find_package(geometry_msgs REQUIRED)
 find_package(ros2_shared REQUIRED)
 find_package(sensor_msgs REQUIRED)
 find_package(std_msgs REQUIRED)
 find_package(tello_msgs REQUIRED)
 
-# Local includes
+# Package includes not needed for CMake >= 2.8.11
 include_directories(
   include
   h264decoder
-  ${camera_calibration_parsers_INCLUDE_DIRS}
-  ${geometry_msgs_INCLUDE_DIRS}
   ${ros2_shared_INCLUDE_DIRS}
-  ${sensor_msgs_INCLUDE_DIRS}
-  ${std_msgs_INCLUDE_DIRS}
   ${tello_msgs_INCLUDE_DIRS})
```

Comment:
doesn't find_package take care of this for ros2_shared and tello_msgs?

Reply:
Maybe these lines cause dependencies to be generated and we want those dependencies for our projects that might change.

Reply:
AFAIK these lines do not change the dependencies. Colcon runs fine when I remove these lines. In fact, so does CLion. So, I removed them.

IIRC CLion can't find the include path when you first add a dependency on a package that is built in the Colcon workspace. Adding a line like this makes it happy. But at some point CLion gets happy again -- maybe after a full Colcon build? -- and we can remove the include. Next time this happens I'll try to take the time to investigate.